dashboard/app/controllers/teacher_feedbacks_controller.rb

```diff
@@ -0,0 +1,27 @@
+class TeacherFeedbacksController < ApplicationController
+  # GET /teacher_feedbacks/new
+  def new
+    @teacher_feedback = TeacherFeedback.new
```

Comment:
You should leverage CanCan for this. Adding `load_and_authorize_resource` to the top of your controller, which will populate the resource `@teacher_feedback` for you and handle auth. See https://github.com/CanCanCommunity/cancancan/wiki/authorizing-controller-actions

Reply:
Same with the other actions.

You will need to add an auth declaration (can / cannot) to the `ability.rb` first for this resource.

Reply:
Also is `new` needed? I don't think this action will be used.